src/js/utils/ServiceUtil.js

```diff
@@ -2,6 +2,7 @@ import {Hooks} from 'PluginSDK';
 import Service from '../structs/Service';
 import VolumeConstants from '../constants/VolumeConstants';
 
+const VIP_ADDRESS = '0.0.0.0';
```

Comment:
It's not clear where this address should come from, so it's currently a constant here.

c.f. [this test](https://github.com/dcos/dcos-ui/pull/539/files#diff-16861817d9d413eb605e14764e062a86R368)